### ISSUE TITLE
Make block query stretch on small

### ISSUE THREAD

**Issue by jacogr** [polkadot-js/apps#823]
![F91D0FD4-CFCB-4A49-9169-4E04A7D2CED1](https://user-images.githubusercontent.com/1424473/54736965-f8f8ef80-4bad-11e9-966b-4ca41499caa5.png)

**Comment by jacogr** [comment]
You probably won’t use query by hash, but for the block number it would be good if the input here can actually take up the available space

**Comment by jacogr** [comment]
Obviously this is one of many, however without flooding, may make sense to start logging and attacking at least some of them. (It is a long road)

**Comment by rossbulat** [comment]
This would expand the width:

Remove .small in `apps/packages/app-explorer/src/BlockQuery/Query.tsx`:
```
- <div className='small' />
```

Add flex grow to actionrow:
`apps/packages/app-storage/src/index.css`:

```
.storage--actionrow {
+    flex: 1 1 100%;
}
```

To restrict max width and center also:

```
.storage--actionrow {
+    margin: 0 auto;
+    max-width: 700px;
}
```

Perhaps these should be for another class,  `.storage--actionrow-head` or just `&.head` for example.

**Comment by jacogr** [comment]
Closed in #848
